udt.ipynb

```diff
@@ -0,0 +1,250 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **Text Classification**\n",
+    "\n",
+    "### **Dataset: CLINC 150**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "text_train_file = \"datasets/clinc_train.csv\"\n",
```

Comment:
can we download this datasets using public url

Reply:
This file is part of https://github.com/ThirdAILabs/Demos/tree/webinar-nov, shouldn't be in this PR. Removing it.